ecr-login/api/client.go

```diff
@@ -145,9 +152,9 @@ func (self *defaultClient) getAuthorizationToken(registryID string) (*Auth, erro
 	if err != nil || output == nil {
 		if err == nil {
 			if registryID == "" {
-				err = fmt.Errorf("Mising AuthorizationData in ECR response for default registry")
+				err = fmt.Errorf("ecr: Mising AuthorizationData in ECR response for default registry")
```

Comment:
Can we simply wrap `err` with `"ecr: "` (or `errors.Wrap(err, "ecr")` if you have it here) when its returned by this func to make these errors consistent?

Reply:
`err` is nil here.  This is sanity-checking that `output` is not nil.

Reply:
Yeah, I see these are used when `err == nil`, however there is another error bubbled up when `err != nil`, which I think should also be prefixed with `ecr: ` as any error from this point is emitted/handled in the context of `ecr: `.

Reply:
`github.com/pkg/errors` was not already a dependency, but I generally like that package so I've added it and used `errors.Wrap`.